### DISCUSSION TITLE
L-4X

### DISCUSSION THREAD

**Discussion by piotrsaweczko** [mariusz-ostoja-swierczynski/tech-controllers#124] [General]
Czy komuś udało się uruchomić integrację z nową listwą L-4X  WIFI (v. 1.0.13) ? na 2.5.0
<img width="1040" alt="Zrzut ekranu 2024-10-18 o 23 12 56" src="https://github.com/user-attachments/assets/985b44fc-91d6-4cb1-a087-687b26539c46">

Debug: https://pastebin.com/5n60yTwL

**Comment by daroga0002** [comment]
Prosba o podeslanie outputu JSON z API jak w https://github.com/mariusz-ostoja-swierczynski/tech-controllers/blob/main/CONTRIBUTING.md#getting-json-data-from-emodul wtedy bedziemy mogli ocenic co jest zle

**Reply by piotrsaweczko** [reply]
Oto wynik: https://pastebin.com/ZSs0bcqt

**Reply by daroga0002** [reply]
to jest wogole dziwne bo masz w `zones` cos takiego:

```json
..................
                },
                "actuators": [],
                "underfloor": {},
                "windowsSensors": [],
                "additionalContacts": []
            },
            null,
            {
                "zone": {
                    "id": 9070,
                    "parentId": 9000,
                    "time": "2024-10-21T08:49:19.234713",
                    "duringChange": false,
                    "index": 4,
                    "currentTemperature": 215,
                    "setTemperature": 215,
                    "flags": {
                        "relayState": "off",
..................
```

gdzie ten `null` jest totalnie od czapy i nie wiadomo czemu API zwraca taka odpowiedz. Czy moze miales jakies inne termostaty zarejestrowane ktorych juz nie masz?? Bo za bardzo nie rozumiem czemu tam ten null wskoczyl

**Reply by piotrsaweczko** [reply]
Okej to już wiem w czym może być rzecz... Mam w sterowniku strefę 1,2,3 (tu jest tylko podłogówka) i na 4 zacisku tj. 4 strefie jest teraz pusto (docelowo trafią tu dwie pętle z 3 strefy - teraz są tam 4 siłowniki podłogówki) ale teraz nic nie jest podłączone. Kolejne termostaty pokojowe są na strefie 5, 6 i 7 (wszystkie to model R-X). 

Czy to może być przyczyną ?

**Reply by daroga0002** [reply]
jezeli to jest juz ustawione na listwie to moze tak byc jak piszesz. Pewnie zeby dobrze to bylo raportowane to trzeba by bylo ta 4 strefe wyrzucic z konfiguracji sterownika i dodac jak juz bedzie jakies urzadzenie faktycznie

**Reply by piotrsaweczko** [reply]
W sterowniku ta 4 strefa nie jest aktywna - nie widać jej tez w emodule.
![Zrzut ekranu 2024-10-21 o 11 32 16](https://github.com/user-attachments/assets/0ddc9afd-b784-4ab3-bca0-6aa06a1997e1)
Zrobiłem inaczej - zamówiłem czujnik temp. przewodowy (bo mam możliwość takiego tam użyć) i podepnę go pod 4 strefę. 
Dam znać czy wszystko zadziała.

**Comment by piotrsaweczko** [comment]
Po dodaniu czujnika do brakującej strefy wszystko zadziałało - dzięki za pomoc!
Potwierdzam działanie z listwą L-4X 👌
![Zrzut ekranu 2024-10-24 o 08 27 18](https://github.com/user-attachments/assets/8d9dc6b4-d83c-4419-a42d-0b686d5dfdde)
